tictactoe.py

```diff
@@ -63,7 +25,7 @@ def play_a_turn(playername, board, first_player = True):
         try:
             player_input = input("Pick a number between 1 to 9 (inclusive).")
             if int(player_input) in range(1,10):
-                break
+                break #why break here? 
```

Comment:
The break at line 66 is to stop asking the player to input a number. It is a while loop that asks player enter an input unless player input a valid number.